sequencer/src/lib.rs

```diff
@@ -368,15 +368,18 @@ where
     }
 
     let epoch_height = genesis.epoch_height.unwrap_or_default();
+    let drb_difficulty = genesis.drb_difficulty.unwrap_or_default();
```

Comment:
this `unwrap_or_default()` is going to get 0 if it's not set. should we do `unwrap_or(SOME_DEFAULT_VALUE)` instead?